src/services/sso.auth.service.js

```diff
@@ -0,0 +1,41 @@
+import * as microsoftTeams from "@microsoft/teams-js";
+
+// An authentication that will only request an access token for the logged in user.
+// This token can then be used to request other resources.
+class SSOAuthService {
+  constructor() {
+    // Initialize the Teams SDK
+    microsoftTeams.initialize();
+
+    // Flag that this service assumes using Teams SSO
+    this.isSSO = true;
+  }
+
+  getUser() {
+    return new Promise((resolve, reject) => {
+      microsoftTeams.authentication.getUser({
```

Comment:
Hmm... This part will always fail :(

Reply:
I wonder if there is a way to decode the auth token and get this information out of it...

Reply:
Oh I can do that...I did that somewhere else.